korge/src/commonMain/kotlin/com/soywiz/korge/newui/UIComboBox.kt

```diff
@@ -5,6 +5,7 @@ import com.soywiz.korge.view.*
 import com.soywiz.korim.color.*
 import com.soywiz.korio.async.*
 
+@Deprecated("Use .ui.uiComboBox instead")
```

Comment:
What about adding `@Deprecated("Use .ui.uiComboBox instead", replaceWith = ReplaceWith("..."))`

Reply:
Hm, good suggestion. I'll try to add this.